use_cases/ecs-ec2-daemon-collector/task-definition-templates/cloudwatch-task-logs.json

```diff
@@ -0,0 +1,83 @@
+{
+    "family": "fluent-bit-daemon-collector-cloudwatch",
+    "networkMode": "host",
+    "taskRoleArn": "{INSERT YOUR TASK ROLE ARN}", 
+    "executionRoleArn": "{INSERT YOUR EXECUTION ROLE ARN}",
+    "volumes": [
+        {
+            "name": "applogs",
+            "host": {
+                "sourcePath": "/var/lib/docker/containers"
+            }
+        },
+        {
+            "name": "fluent-bit-buffer-directory",
+            "host": {
+                "sourcePath": "/var/fluent-bit"
+            }
+        }
+    ],
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "public.ecr.aws/aws-observability/aws-for-fluent-bit:init-stable",
+            "name": "fluent-bit-daemon-collector",
+            "logConfiguration": {
+                "logDriver": "awslogs",
+                "options": {
+                    "awslogs-group": "/aws/ecs/{INSERT CLUSTER_NAME}/collector",
+                    "awslogs-region": "{INSERT REGION}",
+                    "awslogs-create-group": "true",
```

Comment:
I should remove this/set it to false. Log Groups are normally managed by infra as code outside of tasks so that they can be easily cleaned up. 

Also, the [AmazonECSTaskExecutionRolePolicy](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html) does not have create group permissions so if I include this then I require customers to have more perms on their execution or instance role.

Instead, log group can be created with AWS CLI or in the CFN template.